Makefile

```diff
@@ -85,6 +85,10 @@ BASE_GO_PACKAGE = github.com/mongodb/mongodb-atlas-kubernetes/v2
 GO_LICENSES = go-licenses
 DISALLOWED_LICENSES = restricted,reciprocal
 
+# JWT
+JWT_RSA_PEM_KEY_BASE64 ?= "please override rsa key in a one line base64 encoding"
+JWT_APP_ID ?= "please override app id"
```

Comment:
let's keep sane default values here (i.e. empty string) and rather test for non-empty values in the actual script logic.